modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -936,6 +940,7 @@ struct InferList2: public cv::detail::KernelTag {
                     // FIXME: NOW Assume Mats are always BLOBS (not
                     // images)
                     const auto &vec = this_vec.rref<cv::Mat>();
+                    GAPI_Assert(vec[list_idx].depth() == CV_32F);
```

Comment:
What is it for?

Reply:
Removed. CV32F - tensor, not image. But CV8U